Ladybird/Qt/SettingsDialog.cpp

```diff
@@ -28,6 +28,12 @@ SettingsDialog::SettingsDialog(QMainWindow* window)
     m_search_engine_dropdown->setText(qstring_from_ak_string(Settings::the()->search_engine().name));
     m_search_engine_dropdown->setMaximumWidth(200);
 
+    m_preferred_languages = new QLineEdit(this);
```

Comment:
It would be nice if this text box matched the behavior of the `m_new_tab_page` text box by closing the settings dialog when return is pressed.